test/unit/foreman/renderer/snapshots/ProvisioningTemplate/provision/Kickstart_default.rhel9_dhcp.snap.txt

```diff
@@ -0,0 +1,197 @@
+# This kickstart file was rendered from the Foreman provisioning template "Kickstart default".
+# for snapshot-ipv4-dhcp-rhel9 running RHEL 9 x86_64
+# Organization: Organization 1
+# Location: Location 1
+
+
+url --url http://mirror.example.com/rhel/9/os/x86_64
+
+
+lang en_US.UTF-8
+selinux --enforcing
+keyboard us
+
+network --device=00-f0-54-1a-7e-e0 --hostname snapshot-ipv4-dhcp-rhel9 --noipv6 --bootproto dhcp --mtu=1142 --nameserver=192.168.42.2,192.168.42.3
+
+rootpw --iscrypted $1$rtd8Ub7R$5Ohzuy8WXlkaK9cA2T1wb0
+firewall --service=ssh
+authselect --useshadow --passalgo=sha256 --kickstart
+timezone --utc UTC 
+
+services --disabled gpm,sendmail,cups,pcmcia,isdn,rawdevices,hpoj,bluetooth,openibd,avahi-daemon,avahi-dnsconfd,hidd,hplip,pcscd
```

Comment:
Thinking out loud: I wonder how many of these still exist in a current RHEL. Still, probably best to keep them in for older RHEL compatibility.